packages/node_modules/@ciscospark/react-container-read-receipts/src/selectors.js

```diff
@@ -1,14 +1,36 @@
+/* eslint-disable max-params */
 import {createSelector} from 'reselect';
 
 const getActivities = (state) => state.conversation.get(`activities`);
 const getParticipants = (state) => state.conversation.get(`participants`);
+const getCurrentUser = (state) => state.user.get(`currentUser`);
```

Comment:
It looks like currentUser is a normal object. If that's the case then I think this will cause the selector to return a new object every time and force the main selector to rerun every time. May want to just grab `user` and then grab `currentUser` from inside the selector.